Core/include/Acts/TrackFitting/detail/VoidKalmanComponents.hpp

```diff
@@ -45,10 +46,12 @@ struct VoidKalmanUpdater {
   ///
   /// @return The copied predicted parameters
   template <typename track_state_t, typename predicted_state_t>
-  auto operator()(track_state_t& trackState,
-                  const predicted_state_t& predicted) const {
+  Acts::Result<void> operator()(const Acts::GeometryContext& /*gctx*/,
```

Comment:
Why did the return type change by this here?

Reply:
Before it was actually just implicit, it returned whatever it returned, and it was the updated parameters, but I think this was either compiled nowhere, or never checked so far.

The `GainMatrixUpdater` already returned `Result<void>` so I'm effectively only harmonizing the interface.